localization/ndt_scan_matcher/src/util_func.cpp

```diff
@@ -129,7 +129,7 @@ void get_nearest_timestamp_pose(
     const rclcpp::Time pose_time_stamp = output_new_pose_cov_msg_ptr->header.stamp;
     if (pose_time_stamp > time_stamp) {
       // TODO(Tier IV): refactor
-      if (pose_time_stamp.seconds() == 0.0) {
+      if (output_old_pose_cov_msg_ptr->header.stamp.sec == 0.0) {
```

Comment:
@meliketanrikulu Could you please fix it like this?
```suggestion
      if (output_old_pose_cov_msg_ptr->header.stamp.seconds() == 0.0) {
```